blivet/partitioning.py

```
@@ -1158,6 +1159,27 @@ def reserve_request(self):
+class StratisRequest(Request):
+
+    def __init__(self, fs):
+        """
+            :param fs: the stratis filesystem being requested
+            :type fs: :class:`~.devices.StratisFilesystemDevice`
+        """
+        super(StratisRequest, self).__init__(fs)
+        self.base = int(fs.size / LINUX_SECTOR_SIZE)
+
+        if fs.req_grow:
+            limits = [int(l / LINUX_SECTOR_SIZE) for l in
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)